src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -4958,7 +4970,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 12.5;
```

Comment:
Please don't :D

Reply:
Sorry, had that thing in mind the last days… Will be out/in/back/$whatever in a moment